.github/workflows/Validate Ambassadors JSON.yml

```diff
@@ -0,0 +1,60 @@
+name: Validate Ambassadors JSON
+
+on:
+  push:
+    paths:
+      - 'programs/ambassadors/ambassadors.json'
+  pull_request:
+    paths:
+      - 'programs/ambassadors/ambassadors.json'
+
+jobs:
+  validate-json:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node 20
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Install Dependencies
+        run: npm install ajv ajv-formats
+
+      - name: Validate ambassadors.json
+        run: |
+          node -e "
+          const Ajv = require('ajv');
```

Comment:
To run the validation against a 2020-12 schema the dependency to use is: `ajv/dist/2020`

Right now the action is failing:
https://github.com/json-schema-org/community/actions/runs/9631882401